.github/workflows/test.yaml

```diff
@@ -2,32 +2,59 @@ name: Tests
 
 on:
   push:
-    branches:
-      - '*'
+    branches: [ 'main' ]
```

Comment:
Just out of curiosity, is there a specific reason for not having this run on push to branches other than main? I realise it'll still run on PR's to merge to main so it's not a big deal

Reply:
Just followed Kuadrant operator https://github.com/Kuadrant/kuadrant-operator/blob/7dcbc3dabbe13d2092adeaf018b2aaccb0eee23a/.github/workflows/test.yaml#L5

If you want tests to be run every time you push new commits to your branch, we can discuss. I do not see it necessary, so I changed to be the same as the kuadrant operator.

Reply:
Gotcha, seems reasonable, thanks